src/OrchardCore.Modules/OrchardCore.ContentLocalization/Views/ContentCulturePickerRender.cshtml

```diff
@@ -0,0 +1,19 @@
+@using System.Globalization;
```

Comment:
I'm not sure about this name: Do you guys have a convention for cases like this ?

Reply:
`ContentCulturePicker.cshtml` LGTM

Reply:
I already have that shape that is called `ContentCulturePicker.cshtml`. That shape only loads data and render the `ContentCulturePickerRender.cshtml` shape. This allows users to override only the look and feel without having to load the data. Is this a good practice ?

Reply:
ContentCulturePickerContainer